### PR TITLE
Update Kotlin instructions

### PR DESCRIPTION
**Pull request by matklad**
I've checked that this indeed works locally. 

cc @yole

### CODE REVIEW

File: tutorials/kotlin.md [hunk 1]
```diff
@@ -77,4 +74,4 @@ There are a number of [open source Kotlin projects](https://github.com/search?l=
 * [haskell-idea-plugin](https://github.com/atsky/haskell-idea-plugin)
 * [intellij-markdown](https://github.com/valich/intellij-markdown)
 * [IntelliJ-presentation-assistant](https://github.com/chashnikov/IntelliJ-presentation-assistant)
-* [leanpub-plugin](https://github.com/hhariri/leanpub-plugin)
\ No newline at end of file
+* [leanpub-plugin](https://github.com/hhariri/leanpub-plugin)
```
Comment (ignatov) [inline tutorials/kotlin.md]: An additional new line.

Comment (matklad) [inline tutorials/kotlin.md]: Yep, this file lacked the trailing newline, and my editor automatically ensures that there is one. 

What is the convention for this repo? Judging by the surrounding files there isn't any...

Will remove this from the PR.

Comment (matklad) [inline tutorials/kotlin.md]: Fixed!

**Comment by ignatov** [comment]
Thanks

**Pull request merged**